website/docs/r/advanced_cluster.html.markdown

```diff
@@ -181,7 +181,7 @@ resource "mongodbatlas_advanced_cluster" "cluster" {
         node_count    = 1
       }
       provider_name = "GCP"
-      priority      = 0
+      priority      = 5
       region_name   = "US_EAST_4"
```

Comment:
also can you confirm if we need to update priorities on cluster page as well? https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cluster

Reply:
no, the examples have the correct priority